src/radical/pilot/agent/executing/base.py

```diff
@@ -59,16 +62,13 @@ def create(cls, cfg, session):
         from .shell_fs import ShellFS
         from .abds     import ABDS
 
-        try:
-            from .orte     import ORTE
-        except:
-            pass
+      # from .orte     import ORTE
```

Comment:
Are we sure we want to leave around the commented code?

Reply:
It is the same scheme we used in other, similar parts of the code - see https://github.com/radical-cybertools/radical.pilot/blob/devel/src/radical/pilot/agent/lm/base.py#L120 for example.